packages/server/graphql/public/mutations/createTeamPromptResponse.ts

```diff
@@ -0,0 +1,55 @@
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {upsertTeamPromptResponse} from '../../../postgres/queries/upsertTeamPromptResponses'
+import {getUserId, isTeamMember} from '../../../utils/authorization'
+import publish from '../../../utils/publish'
+import standardError from '../../../utils/standardError'
+import {MutationResolvers} from '../resolverTypes'
+
+const createTeamPromptResponse: MutationResolvers['createTeamPromptResponse'] = async (
+  _source,
+  {meetingId, userId},
+  {authToken, dataLoader, socketId: mutatorId}
+) => {
+  const viewerId = getUserId(authToken)
+  const operationId = dataLoader.share()
+  const subOptions = {mutatorId, operationId}
+
+  // AUTH
+  if (userId !== viewerId) {
+    return standardError(new Error("Can't create response for others"), {userId: viewerId})
+  }
+  const meeting = await dataLoader.get('newMeetings').load(meetingId)
+  const {endedAt, teamId} = meeting
+  if (!isTeamMember(authToken, teamId)) {
+    return standardError(new Error('Team not found'), {userId: viewerId})
+  }
+  if (endedAt) return standardError(new Error('Meeting already ended'), {userId: viewerId})
+
+  // VALIDATION
+  /**
+   * TODO: check if there's already a response for this user in this meeting
```

Comment:
To check this, we need `getTeamPromptResponsesByMeetingId` query, which I have in [subsequent PR](https://github.com/ParabolInc/parabol/pull/6388/files#diff-00749d0652800c424824c0c23172bbae293db5aa559f1fd57d8295a999656210). To prevent this PR becomes a monster, I'll save this for later.

Reply:
This issue is addressed [here](https://github.com/ParabolInc/parabol/pull/6388/files#diff-432ce08d5f6c923e8425643ee15814afda231e2941dcb4a278e99a87824e02deR33)